webauthn-rs-core/src/core.rs

```diff
@@ -50,6 +50,10 @@ use base64urlsafedata::Base64UrlSafeData;
 /// as much as possible.
 #[derive(Debug)]
 pub struct WebauthnCore {
+    /// The RP name
+    #[cfg(debug_assertions)]
+    pub rp_name: String,
```

Comment:
Could just be worth making a getter instead of the debug assertions.

Reply:
done!